packages/runtime/test/integration/atomic-transfer.ts

```diff
@@ -131,6 +133,7 @@ describe("Algorand Smart Contracts - Atomic Transfers", function () {
         fromAccount: john.account,
         appID: appID,
         payFlags: { totalFee: 1000 }
+
```

Comment:
why do we add empty lines?

Reply:
must be a find and replace error, will remove it